content/kots-cli/download.md

```diff
@@ -5,4 +5,24 @@ title: kots download
 weight: 90120
 ---
 
-Once an application is running and has the Admin Console (kotsadm) deployed with it, you can run `kots download` to retrieve a copy of the application manifests from the cluster, and store them in a specific directory structure on your workstation.
+Retrieves a copy of the application manifests from the cluster, and store them in a specific directory structure on your workstation. Requires a running KOTS application with the Admin Console (kotsadm).
+
+### Usage
+```bash
+kubectl kots download [app-slug] [flags]
+```
+* _Replace `[app-slug]` with the "slug" of your KOTS application_ 
+* _Provide `[flags]` according to the table below_ 
+
+| Flag                 | Type | Description |
+|:----------------------|------|-------------|
+| `--dest` | string |        the directory to store the application in (default "/Users/austin") |
```

Comment:
This defaults to your home directory ($HOME var for Mac and Linux, $USERPROFILE dir on Windows). I think that this doc should not contain your homedir, but an indicator that this default is environment specific.

Reply:
I'll go ahead and add the helpful details about the flag. If we change it here, we may also want to change what is displayed for --help.

Reply:
Saying (defaults to $HOME).